opensearch/src/main/java/org/opensearch/sql/opensearch/client/OpenSearchRestClient.java

```diff
@@ -46,6 +47,26 @@ public class OpenSearchRestClient implements OpenSearchClient {
   /** OpenSearch high level REST client. */
   private final RestHighLevelClient client;
 
+  @Override
+  public boolean isExist(String indexName) {
```

Comment:
```suggestion
  public boolean exists(String indexName) {
```

would be more conventional in Java.